### DISCUSSION TITLE
Metrics for File Descriptor (open file)

### DISCUSSION THREAD

**Discussion by rareman99** [n27051538/solaris_exporter#17] [General]
Hi,
Do you have metrics for File Descriptor (open file) or metric for collect sysconfig (example /etc/system, /etc/project)?

**Comment by n27051538** [comment]
Hello,
No. And no plans to add something.

ср, 14 дек. 2022 г., 10:40 rareman99 ***@***.***>:

**Reply by rareman99** [reply]
Thanks
